plugins/bn-avro/qml/qml.pro

```diff
@@ -0,0 +1,19 @@
+TOP_BUILDDIR = $$OUT_PWD/../../..
+TOP_SRCDIR = $$PWD/../../..
+
+include($${TOP_SRCDIR}/config.pri)
+
+TARGET = dummy
+TEMPLATE = lib
+
+lang_avro.path = "$$UBUNTU_KEYBOARD_LIB_DIR/avro/"
```

Comment:
this should be bn-avro aswell

Reply:
@Fuseteam, Do I need `lang_avro` to be changed `lang_bn-avro` too?